### ISSUE TITLE
Check:imports command wrong reports when we call the value of a case enum

### ISSUE THREAD

**Issue by alikhosravidev** [imanghafoori1/php_token_analyzer#19]
PHP: v8.1
Laravel: v9
microscope: v1.0.318
Os: Pop Os

Autoload psr4:
"autoload": { "files": [ "app/helpers.php" ], "psr-4": { "App\\": "app/", "Database\\Factories\\": "database/factories/", "Database\\Seeders\\": "database/seeders/" } },

```php
<?php

namespace App\Core\Notification\Criteria\Notification;

use Illuminate\Database\Eloquent\Builder;
use App\Core\Notification\Enums\NotificationVisibility;
use Prettus\Repository\Contracts\CriteriaInterface;
use Prettus\Repository\Contracts\RepositoryInterface;

class PublicVisibilityCriteria implements CriteriaInterface
{
    /**
     * Apply criteria in query repository.
     *
     * @param Builder $model
     */
    public function apply($model, RepositoryInterface $repository): mixed
    {
        return $model->whereRelation('notification', 'visibility', NotificationVisibility::PUBLIC->value);
    }
}

```

Reported message:

1 Class does not exist:
App\Core\Notification\Criteria\Notification\value
at app/Core/Notification/Criteria/Notification/PublicVisibilityCriteria.php:19

**Comment by imanghafoori1** [comment]
this is due to the fact that the case is named "public" and it is fixed by now.
